variables.tf

```
@@ -17,6 +17,7 @@ variable "ec2_stop_scheduler" {
     cron_expression = string
     instance_ids    = list(string)
     description     = optional(string, "")
+    patching_dates  = optional(list(string), [])
```

```suggestion
    exception_dates  = optional(list(string), [])
```

eventually using a more generic name for the exception of stop dates....